### PR TITLE
After zoom points position are wrong

### PR DESCRIPTION
**Pull request by maloun96**
Solve problem: https://github.com/apache/superset/issues/12201

BEFORE
Kapture 2021-01-06 at 16 09 40

AFTER
Kapture 2021-01-06 at 15 50 44

### CODE REVIEW

**Review by rusackas — APPROVED** [review]
LGTM!

File: build/nv.d3.js
Comment (villebro) [inline build/nv.d3.js:181]: I wonder why this is shown as changed - on quick glance these are identical.

File: build/nv.d3.js
Comment (villebro) [inline build/nv.d3.js:10003]: When looking at https://github.com/novus/nvd3/pull/2054, I can't see this having any harmful side-effects if the new changes "just work". So I'd be inclined to merge this change. We can always revert later if it does break something else, but in the absence of any known defect this looks fine to me.

**Review by villebro — COMMENTED** [review]
A few comments

File: build/nv.d3.js
Comment (rusackas) [reply build/nv.d3.js:10003]: I came to the same conclusion... this seems like it'll help more than it _might_ hurt.

**Comment by rusackas** [comment]
@mistercrunch can you please merge and release this when you have the chance, and/or give @villebro and I write access to this repo and on NPM?

**Review by mistercrunch — APPROVED** [review]

**Comment by rusackas** [comment]
Thanks for the research and work on this! This has been cherry picked over to [the main repo that gets published to npm](https://github.com/apache-superset/nvd3), and [this PR](https://github.com/apache-superset/superset-ui/pull/891) has been opened on Superset UI to pull in that new version. Once that's merged, we'll bump Superset UI package version(s) and pull those in with a new PR on Superset. Easy! ;)

**Pull request merged**
